### PR TITLE
Add package tags, fix description typos

### PR DESCRIPTION
**Pull request by dschubertpxc**

### CODE REVIEW

File: src/Moryx.Communication.Serial/Moryx.Communication.Serial.csproj [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>MORYX binary connection for serial communication</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>iot;iiot;industrial</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.Communication.Serial/Moryx.Communication.Serial.csproj:8]: What about moryx here?

File: src/Moryx.Container/Moryx.Container.csproj [hunk 2]
```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>MORYX container functionality based on Castle.Windsor.</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>windsor;castle;iiot;moryx</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.Container/Moryx.Container.csproj:8]: container, ioc?

e.g. "MORYX;Container;IoC"

File: src/Moryx.Model.InMemory/Moryx.Model.InMemory.csproj [hunk 3]
```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>InMemory extension for unit tests referencing Moryx.Model</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>ef6;Entity Framework;moryx;iiot;inmemory;iiot;iot</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.Model.InMemory/Moryx.Model.InMemory.csproj:8]: You use `Entity Framework` sometimes with spaces and sometimes without

e.g. "MORYX;Entity Framework;Data Model;Database;InMemory"

File: src/Moryx.Runtime.SmokeTest/Moryx.Runtime.SmokeTest.csproj [hunk 4]
```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Runtime RunMode for executing smoke tests.</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>command line parser;moryx;tests;iiot;iot</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.Runtime.SmokeTest/Moryx.Runtime.SmokeTest.csproj:8]: command line parser should be not used as tag here. This class library is used for smoke tests runtime environment

e.g.: "MORYX;Runtime;Test;SmokeTests"

File: src/Moryx.Runtime.Wcf/Moryx.Runtime.Wcf.csproj [hunk 5]
```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>WebService integration for MORYX applications based on WCF</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>webservice;web;wcf;communication;castle;moryx;iiot;iot</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.Runtime.Wcf/Moryx.Runtime.Wcf.csproj:8]: This class library is used for wcf integration into runtime environment

maybe: "MORYX;Runtime;Server;Client;WCF;communication"

File: src/Moryx.Runtime.WinService/Moryx.Runtime.WinService.csproj [hunk 6]
```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Provides runtime mode for running as Windows Service.</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>windows service;command line parser;moryx;iiot;iot</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.Runtime.WinService/Moryx.Runtime.WinService.csproj:8]: command line parser should be not used as tag here

maybe: "MORYX;Runtime;Server;Windows Service"

File: src/Moryx.Runtime/Moryx.Runtime.csproj [hunk 7]
```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Server side component types and implementations for MORYX applications</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>serverside;server;moryx;runtime;iiot;iot</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.Runtime/Moryx.Runtime.csproj:8]: e.g. just "MORYX;Runtime;Server"?

File: src/Moryx.TestTools.SystemTest/Moryx.TestTools.SystemTest.csproj [hunk 8]
```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Library with helper classes for SystemTests.</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>newtonsoft;json;maintenance;helper class;moryx;iiot;iot</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.TestTools.SystemTest/Moryx.TestTools.SystemTest.csproj:8]: e.g. "MORYX;Tests;SystemTests"

File: src/Moryx.TestTools.UnitTest/Moryx.TestTools.UnitTest.csproj [hunk 9]
```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Library with helper classes for UnitTests.</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>castle;windsor;helper class;unit test;moryx;iiot;iot</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.TestTools.UnitTest/Moryx.TestTools.UnitTest.csproj:8]: e.g. "MORYX;Tests;UnitTests"

File: src/Moryx.Tools.Wcf/Moryx.Tools.Wcf.csproj [hunk 10]
```diff
@@ -3,8 +3,9 @@
   <PropertyGroup>
     <TargetFrameworks>net45;net461</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <Description>Types and tools for WCF integation in MORYX server or client applications</Description>
-    <CreatePackage>true</CreatePackage>
+    <Description>Types and tools for WCF integration in MORYX server or client applications</Description>
+    <CreatePackage>true</CreatePackage> 
+    <PackageTags>http;wcf;server;client;service model;moryx;iiot;iot</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.Tools.Wcf/Moryx.Tools.Wcf.csproj:8]: e.g. "MORYX;WCF;Service Model"

File: src/Moryx.Runtime.Maintenance/Moryx.Runtime.Maintenance.csproj [hunk 11]
```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Core module to configure and control MORYX based applications</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>core;module;moryx;iiot;iot</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.Runtime.Maintenance/Moryx.Runtime.Maintenance.csproj:8]: e.g. "MORYX;Runtime;Server;Configuration;UI"

File: src/Moryx.Runtime.Kernel/Moryx.Runtime.Kernel.csproj [hunk 12]
```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Kernel components that comprise the MORYX server side runtime environment</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>command line parser;logging;moryx;server;iiot;iot</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.Runtime.Kernel/Moryx.Runtime.Kernel.csproj:8]: command line parser should not be used here

e.g. "MORYX;Runtime;Server"

File: src/Moryx.Communication.Serial/Moryx.Communication.Serial.csproj (hunk 1 above)
Comment (dbeuchler) [reply src/Moryx.Communication.Serial/Moryx.Communication.Serial.csproj:8]: e.g. "MORYX;Communication;Serial;Ports"

File: src/Moryx.Model.PostgreSQL/Moryx.Model.PostgreSQL.csproj [hunk 13]
```diff
@@ -3,8 +3,9 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.1;net45;net461;net5.0</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <Description>Adapter for Mory.Model on PostgreSQL</Description>
+    <Description>Adapter for Moryx.Model on PostgreSQL</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>ef6;npgsql;postgres;postgresql;adapter;model;moryx;iiot;iot</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.Model.PostgreSQL/Moryx.Model.PostgreSQL.csproj:8]: e.g. "MORYX;Entity Framework;Data Model;Database;Npgsql;PostgreSQL;Postgres"

File: src/Moryx.Model/Moryx.Model.csproj [hunk 14]
```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Datamodel integration for MORYX applications based on Entity Framework</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>EntityFramework;entity;datamodel;data;model;moryx;iiot;reflection;iiot;iot</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.Model/Moryx.Model.csproj:8]: e.g. "MORYX;Entity Framework;Data Model;Database"

File: src/Moryx.Runtime.DbUpdate/Moryx.Runtime.DbUpdate.csproj [hunk 15]
```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Runtime RunMode for executing database migrations.</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>database;migration;runtime;moryx;iiot;iot</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx.Runtime.DbUpdate/Moryx.Runtime.DbUpdate.csproj:8]: e.g. "MORYX;Runtime;Entity Framework;Data Model;Database;Migration"

File: src/Moryx/Moryx.csproj [hunk 16]
```diff
@@ -6,6 +6,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Description>Core package of the MORYX ecosystem. It defines the necessary types for MORYX compatibility as well as commonly required tools.</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>json;modularity;IoT;IIoT;industrial;IoT framework;newtonsoft;foundation</PackageTags>
```
Comment (dbeuchler) [inline src/Moryx/Moryx.csproj:9]: do not use newtonsoft here, instead e.g. our core namespaces

e.g.: "MORYX;Serialization;Configuration; ......."

**Review by dbeuchler — COMMENTED** [review]
My review comments are some ideas for the packages. I dont know where I have read this but in general as less tags as possible should be used. The tags should be as precise as possible

File: src/Moryx.Model.InMemory/Moryx.Model.InMemory.csproj (hunk 3 above)
Comment (Toxantron) [reply src/Moryx.Model.InMemory/Moryx.Model.InMemory.csproj:8]: Duplicate iiot tag

**Comment by Toxantron** [comment]
Thanks for adding the tags. Which impression did you get about best practices for tags on nuget.org?

**Comment by dschubertpxc** [comment]
> Thanks for adding the tags. Which impression did you get about best practices for tags on nuget.org?

Thanks for the reviews @dbeuchler @Toxantron. Some best practice points I'm taking away:

- Use tags with upper and lower case
- Use tags as precise as possible for specific package and use few as possible (as @dbeuchler mentioned).
- Use tags that are matching the core namespaces if appropriate
- Preferably don't tag the names of reference packages directly, but may include their functionality instead

**Review by dbeuchler — APPROVED** [review]

**Pull request merged**